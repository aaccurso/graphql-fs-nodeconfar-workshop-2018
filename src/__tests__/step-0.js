const { graphql } = require('graphql');
const schema = require('../schema');

describe('step 0', () => {
  it('should query hello', async () => {
    const source = `
      query hello {
        hello
      }
    `;

    expect(await graphql({ schema, source })).toMatchSnapshot();
  });

  it('should query hello nodeconf', async () => {
    const source = `
      query hello {
        hello(name: "NodeConf")
      }
    `;

    expect(await graphql({ schema, source })).toMatchSnapshot();
  });
});
