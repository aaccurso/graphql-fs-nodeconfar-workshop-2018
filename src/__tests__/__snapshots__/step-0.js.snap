// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`step 0 should query hello 1`] = `
Object {
  "data": Object {
    "hello": "Hello World!",
  },
}
`;

exports[`step 0 should query hello nodeconf 1`] = `
Object {
  "data": Object {
    "hello": "Hello NodeConf!",
  },
}
`;
